spec/features/cart/visitor_can_modify_cart_spec.rb

```diff
@@ -35,14 +35,14 @@
       click_on "Add 1"
       expect(page).to have_content("2")
       expect(page).to have_content("Quantity: 2")
-      expect(page).to have_content("Subtotal: 19.98")
```

Comment:
We might want to see how we can refactor to take out these hard-coded values, but seems fine for now.